### PR TITLE
 new(driver): update exit events `PPME_SOCKET_SOCKET_X` with enter params

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area driver-kmod

/area driver-bpf

/area driver-modern-bpf

/area libscap-engine-gvisor

/area libscap

/area libsinsp

/area tests

**Does this PR require a change in the driver versions?**

/version driver-SCHEMA-version-major

**What this PR does / why we need it**:

This PR is part of https://github.com/falcosecurity/libs/pull/2068. 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: driver/modern_bpf/definitions/events_dimensions.h [hunk 1]
```diff
@@ -89,7 +89,7 @@
 #define CAPSET_E_SIZE HEADER_LEN
 #define CAPSET_X_SIZE HEADER_LEN + sizeof(int64_t) + sizeof(uint64_t) * 3 + PARAM_LEN * 4
 #define SOCKET_E_SIZE HEADER_LEN + sizeof(uint32_t) * 3 + PARAM_LEN * 3
-#define SOCKET_X_SIZE HEADER_LEN + sizeof(int64_t) + PARAM_LEN
+#define SOCKET_X_SIZE HEADER_LEN + sizeof(int64_t) + sizeof(uint32_t) * 3 + PARAM_LEN * 4
```
Comment (FedeDP) [inline driver/modern_bpf/definitions/events_dimensions.h:92]: I am still looking for a way to automatically declare these ones :rofl: perhaps some `constexpr` magic?

**Review by FedeDP — DISMISSED** [review]
/approve

**Review by FedeDP — DISMISSED** [review]
/approve

**Review by FedeDP — APPROVED** [review]
/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**